robot/rover/science/raman.ino

```diff
@@ -6,10 +6,9 @@
 #include <include/commands/CommandCenter.h>
 #include <include/commands/ScienceCommandCenter.h>
 #include "include/Carousel.h"
-#include <cstdint>
 
-const uint8_t NUMBER_OF_STOPPABLES = 5;
-const uint8_t NUMBER_OF_UPDATABLES = 5;
+const int NUMBER_OF_STOPPABLES = 5;
+const int NUMBER_OF_UPDATABLES = 5;
```

Comment:
any reason for this? We're planning on using strong types everywhere, especially for comms

Reply:
(bad commit somehow made it to the PR)